samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -1536,6 +1565,69 @@ static int cmd_pd(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_fixed_id(const struct shell *shell, size_t argc, char **argv)
+{
+	uint8_t big_index;
+
+	if (argc < 3) {
+		shell_error(
+			shell,
+			"Usage: bct fixed_id <0/1> <BIG index> [broadcast_id in hex (3 octets)]");
+		return -EINVAL;
+	}
+
+	if (!is_number(argv[1])) {
+		shell_error(shell, "Fixed ID must be a digit");
+		return -EINVAL;
+	}
+
+	uint8_t fixed_id = (uint8_t)atoi(argv[1]);
```

Comment:
Reconsider naming. This looks like the actual ID on first glance.

Reply:
Just `fixed`?

Reply:
Did you discuss? I'm also mixing this with the actual ID (noted some alternatives in other comments)